python/pyspark/sql/dataframe.py

```diff
@@ -2238,6 +2238,140 @@ def cube(self, *cols: "ColumnOrName") -> "GroupedData":  # type: ignore[misc]
 
         return GroupedData(jgd, self)
 
+    def unpivot(
```

Comment:
Let'd document it under https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.sql/dataframe.rst

Reply:
https://github.com/apache/spark/pull/37354